src/tests/functional/frame_tests.js

```diff
@@ -82,6 +82,19 @@ test("test following a link sets the frame element's [src]", async ({ page }) =>
   assert.equal(src.searchParams.get("key"), "value", "[src] attribute encodes query parameters")
 })
 
+test("test following a link doesn't set the frame element's [src] if the link has [data-turbo-stream]", async ({ page }) => {
+  await page.goto("/src/tests/fixtures/form.html")
```

Comment:
Unknown if this relates to the test failures, but I think this is already done in the beforeEach hook

Reply:
NVM, frames vs forms 🤦🏻